src/Commands/MeltorCommand.php

```diff
@@ -0,0 +1,309 @@
+<?php
+
+namespace Meltor\Commands;
+
+use Exception;
+use Illuminate\Console\Command;
+use Illuminate\Database\ConnectionInterface;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\DB;
+use Meltor\Meltor;
+
+class MeltorCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'meltor:generate
+                {--f|force               : Execute without confirmation. }
```

Comment:
already exists in all commands as `-n, --no-interaction  Do not ask any interactive question`